GetCertificate/certificate.headerInfo.ts

```diff
@@ -0,0 +1,78 @@
+import { PreferredLanguageEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/PreferredLanguage";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { pipe } from "fp-ts/lib/function";
+
+import * as O from "fp-ts/lib/Option";
+import { match } from "ts-pattern";
+import { HeaderInfo } from "../generated/definitions/HeaderInfo";
+
+import {
+  DefaultLanguage,
+  isSupportedLanguage,
+  SupportedLanguage
+} from "../utils/conversions";
+
+import {
+  Certificates,
+  RecoveryCertificate,
+  TestCertificate,
+  VacCertificate
+} from "./certificate";
+
+const EUROPEAN_LOGO_ID = "1" as NonEmptyString;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const ITALIAN_LOGO_ID = "2" as NonEmptyString;
+
+/* 
+Title and subtitle of certificate header 
+for each supported languageF
+*/
+
+const standardTitleAndSubtitle: {
```

Comment:
Mybe it's overkill, but consider to move those texts into a language-dedicated file like we dod for every text.

Reply:
You're right, but I would leave it to a future refactoring that includes all translated strings